### PR TITLE
Add VecDB integration

### PR DESCRIPTION
**Pull request by jtwinrelevanceai**

### CODE REVIEW

File: langchain/vectorstores/vecdb.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        self.client = vecdb.Client(os.getenv("VECDB_API_KEY"))
+        self.dataset = self.client.create_dataset(dataset_id)
+
+    def add_texts(
+        self,
+        texts: Iterable[str],
+        metadatas: Optional[List[dict]] = None,
+        **kwargs: Any,
+    ) -> List[str]:
+        self.dataset.insert(data=texts, metadata=metadatas, **kwargs)
```
Comment (dev2049) [inline langchain/vectorstores/vecdb.py:25]: this method is expected to return a list of string ids. what's the output of dataset.insert?

File: tests/integration_tests/vectorstores/test_vecdb.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
+"""Test Chroma functionality."""
```
Comment (hwchase17) [inline tests/integration_tests/vectorstores/test_vecdb.py:1]: naming is off, this is not chroma

File: langchain/vectorstores/vecdb.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"""Wrapper around ChromaDB embeddings platform."""
```
Comment (hwchase17) [inline langchain/vectorstores/vecdb.py:1]: naming

**Review by hwchase17 — COMMENTED** [review]
notebook could use some work as well, prints out far too much and has some comments saying pinecone

**Pull request closed without merging**